packages/compass-aggregations/src/modules/auto-preview.ts

```diff
@@ -12,13 +11,15 @@ export type AutoPreviewToggledAction = {
   type: ActionTypes.AutoPreviewToggled;
   value: boolean;
 };
+export type AutoPreviewAction = AutoPreviewToggledAction;
+export type AutoPreviewState = boolean | undefined;
 
-export const INITIAL_STATE = true;
+export const INITIAL_STATE: AutoPreviewState = true;
 
 export default function reducer(
-  state = INITIAL_STATE,
-  action: AnyAction
```

Comment:
Redux [strongly recommends](https://redux.js.org/usage/usage-with-typescript#avoid-action-type-unions) not to use union types for actions (this is why we use `isAction` helpers in other reducers)